docs/user-guide/cli-using-using-team-profiles.md

```diff
@@ -46,8 +46,14 @@ You can convert all of your Zowe CLI and Zowe CLI plug-ins V1 profiles to team p
 ```
 zowe config convert-profiles
 ```
+:::caution
+
+You can continue using Zowe CLI and Zowe CLI plug-ins V1 profiles with Zowe CLI V2. However, we highly recommend that you implement V2 profiles with Zowe CLI V2.
```

Comment:
"... using Zowe CLI and Zowe CLI plug-ins V1 profiles with ..." might read better if "and Zowe CLI plugins" is taken out. The rest looks fine.

Reply:
done -- thank you!